tests/basic_sanity_test.go

```diff
@@ -1,62 +1,63 @@
 package tests_test
 
 import (
-	"flag"
 	"fmt"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
 	"kubevirt.io/containerized-data-importer/tests"
+	"kubevirt.io/containerized-data-importer/tests/framework"
 )
 
 const (
 	TestSuiteName = "Basic Sanity"
 )
 
 var _ = Describe(TestSuiteName, func() {
-	flag.Parse()
+	f := framework.NewFramework("sanity")
+	fmt.Fprintf(GinkgoWriter, "\n******** ns=%+v\n", f.Namespace)
 
-	Describe("CDI service account should exist", func() {
+	Context("CDI service account should exist", func() {
 		It("Should succeed", func() {
-			result, err := tests.RunKubectlCommand("get", "sa", "cdi-sa", "-n", tests.CDIInstallNamespace)
+			result, err := tests.RunKubectlCommand(f, "get", "sa", "cdi-sa", "-n")
```

Comment:
I think you accidentally remove the namespace here, and that is highly likely the source of the failure you are seeing.

Reply:
I think it should be this no?
result, err := tests.RunKubectlCommand(f, "get", "sa", "cdi-sa", "-n", f.CdiInstallNs)